tests/devapps/Net5TestApp/Program.cs

```diff
@@ -1,8 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Don't test files need a license too? Or is it only needed for what we distribute?

Reply:
Afaik everything.